bin/utils/getconfig.py

```diff
@@ -0,0 +1,87 @@
+#!/usr/bin/env python3
+
+#  Copyright (C) 2021 RidgeRun, LLC (http://www.ridgerun.com)
+#  Authors: Daniel Chaves <daniel.chaves@ridgerun.com>
+#           Marisol Zeledon <marisol.zeledon@ridgerun.com>
+#  Based on Texas Instruments Incorporated - http://www.ti.com/
+
+import argparse
+import curses
+import cv2
+import logging
+import numpy as np
+import os
+import signal
+import stat
+import sys
+from time import time
+from time import sleep
+import threading
+import yaml
```

Comment:
You aren't using:
argparse, curses, cv2, numpy, os, signal, stat, time, threading